go.mod

```diff
@@ -49,8 +49,8 @@ require (
 
 replace github.com/gogo/protobuf => github.com/ElrondNetwork/protobuf v1.3.2
 
-replace github.com/ElrondNetwork/arwen-wasm-vm/v1_2 v0.0.0-20210729074338-530860a14ea8 => github.com/ElrondNetwork/arwen-wasm-vm v1.2.28
+replace github.com/ElrondNetwork/arwen-wasm-vm/v1_2 v1.2.28 => github.com/ElrondNetwork/arwen-wasm-vm v1.2.29-0.20210729145932-3dbaf2426369
```

Comment:
I think it will work only if you remove the commit hashes. Let's use just the release tags.